sourcecode/weatherByYear/weatherman.py

```diff
@@ -0,0 +1,26 @@
+from sourcecode.filehandling import FileHandling
+from sourcecode.report_generator import ReportGenerator
+from sourcecode.weather_record_parser import WeatherRecordParser
+
+
+class WeatherManYear(FileHandling):
+    def __init__(self) -> None:
```

Comment:
Do we really need to inherit this on file handling? File Handling should be a separate abstraction which can read and give data to anything using it.